modules/pxp-module-puppet

```diff
@@ -70,7 +70,8 @@ end
 
 def force_unicode(s)
   # Puppet still returns UTF-8 when locale is C or POSIX, so force encoding to that.
-  if s.encoding == Encoding::ASCII_8BIT
+  if !Puppet.features.microsoft_windows? &&
+    (s.encoding == Encoding::US_ASCII || s.encoding == Encoding::ASCII_8BIT)
```

Comment:
It looks like this will change the behavior if `s.encoding == Encoding::ASCII_8BIT` on Windows. From the comment thread, I had the impression the intention was not to change Windows but tbh I'm out of my depth wrt Encoding on Windows. Is this change on Windows what we want here?